routes/routeStack.js

```diff
@@ -43,7 +43,6 @@ const screens = {
         screen: PreferenceMenu,
         navigationOptions: {
             headerShown: false,
-            ...TransitionPresets.ModalSlideFromBottomIOS
```

Comment:
why did you remove this?

Reply:
Good catch! It wasn't supposed to removed